plone/app/relationfield/tests.zcml

```diff
@@ -0,0 +1,16 @@
+<configure
```

Comment:
You can register a behavior by registering the adapters yourself, see https://github.com/plone/plone.behavior/blob/master/plone/behavior/behaviors.rst 
or by by calling https://github.com/plone/plone.behavior/blob/master/plone/behavior/metaconfigure.py#L65 behaviorDirective (should work with _context = portal iirc)